introduction_to_amazon_algorithms/jumpstart-foundation-models/llama3.1-fine-tuning.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Please add one more paragraph after this</p><p> </p><p> </p><p><strong>Note</strong>. The fine-tuning model artifacts for 405B fine-tuning is in its original precision bf16. After QLoRA fine-tuning, we conducted fp8 quantization (<a href="https://github.com/huggingface/transformers/blob/main/docs/source/en/quantization/fbgemm_fp8.md" target="_blank">code source</a>) on the trained model artifacts in bf16 to make it deployable on single <code>ml.p5.48xlarge</code> instance.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Sure</p>